config/docker-compose-secondaries/example-redis.yml

```diff
@@ -0,0 +1,5 @@
+version: '3'
+services:
+  redis:
+    image: "redis:alpine"
+    restart: always
```

Comment:
Do we need this example file? It doesn't seem relevant to our/your needs.

Reply:
It is the example 'secondary image' that I uploaded to OTA Connect. No strict need for it to be present, but it was useful to have something for testing.

Reply:
Oh, I see! That's simpler than I expected, sorry.

Reply:
np. I'll get some documentation in place before merging, but I'd like to make sure this is on the right path first.